docs/README.md

```diff
@@ -1,106 +1,131 @@
-  development environments easier
+- **Increased security**
+  - Centralize source code and other data onto private servers or cloud services instead of local developers' machines.
+  - Provision and manage users with [SSO](https://coder.com/docs/admin/auth) and [Role-based access controlled (RBAC)](https://coder.com/docs/admin/rbac).
 
-- **Increase security:** Centralize source code and other data onto private
-  servers or cloud services instead of local developer machines
+- **Improved compatibility**
+  - Remote workspaces can share infrastructure configurations with other
+  development, staging, and production environments, reducing configuration
+  drift.
 
-- **Improved compatibility:** Remote workspaces share infrastructure
-  configuration with other development, staging, and production environments,
-  reducing configuration drift
+- **Improved accessibility**
+  - Connect to remote workspaces via browser-based IDEs or remote IDE
+  extensions to enable developers regardless of the device they use, whether
+  it's their main device, a lightweight laptop, Chromebook, or iPad.
 
-- **Improved accessibility:** Devices such as lightweight notebooks,
-  Chromebooks, and iPads can connect to remote workspaces via browser-based IDEs
-  or remote IDE extensions
+Read more about why organizations and engineers are moving to remote
+development on [our blog](https://coder.com/blog), the
+[Slack engineering blog](https://slack.engineering/development-environments-at-slack),
+or from [OpenFaaS's Alex Ellis](https://blog.alexellis.io/the-internet-is-my-computer/).
 
 ## Why Coder
 
-The key difference between Coder OSS and other remote IDE platforms is the added
-layer of infrastructure control. This additional layer allows admins to:
+The key difference between Coder and other remote IDE platforms is the added
+layer of infrastructure control.
+This additional layer allows admins to:
 
-- Support ARM, Windows, Linux, and macOS workspaces
-- Modify pod/container specs (e.g., adding disks, managing network policies,
-  setting/updating environment variables)
-- Use VM/dedicated workspaces, developing with Kernel features (no container
-  knowledge required)
+- Simultaneously support ARM, Windows, Linux, and macOS workspaces.
+- Modify pod/container specs, such as adding disks, managing network policies, or
+  setting/updating environment variables.
+- Use VM or dedicated workspaces, developing with Kernel features (no container
+  knowledge required).
 - Enable persistent workspaces, which are like local machines, but faster and
-  hosted by a cloud service
+  hosted by a cloud service.
+
+## How much does it cost?
+
+Coder is free and open source under
+[GNU Affero General Public License v3.0](https://github.com/coder/coder/blob/main/LICENSE).
+All developer productivity features are included in the Open Source version of
+Coder.
+An [Enterprise license is available](https://coder.com/pricing) for enhanced
```

Comment:
@stirby should we keep having the word Enterprise?

cc: @mattvollmer

Reply:
+ should I add anything to this section that highlights some of the Enterprise features?

Reply:
Let's call it Premium in the Restructure.

And instead of 

```
https://coder.com/pricing
```

Send them to 

```
https://coder.com/pricing#compare-plans
```